sql/2021-05-07-update-janelia-information.sql

```diff
@@ -0,0 +1,337 @@
+UPDATE
+    submissions
+SET
+    information_ds_software_version = '3.9.0',
+    information_ds_storage_type = 'NAND',
```

Comment:
I was looking at the other entries in the database, and *none* of them are using "NAND" as the storage type, and this also isn't an option on the info-creator page.  I *do* think that it is useful to distinguish NAND flash from 3DXP  (Optane) storage, but we should make this consistent with the data collected from the info-creator, so that the content in the database is getting more consistent over time instead of less consistent.  Should we use "SSD" here as a proxy for "NAND", or should we bulk-convert "SSD" in the database to "NAND"?

Currently, the info-creator gives options for "Storage Media type" = "Memory, SSD, HDD, NVM, Other", and "Storage Media interface" = "USB2/3, SATA/2/3, NVMe, NVDIMM, PCIe2/3/4x".

Reply:
We discussed today to change it to "Flash", also cause there is other types/variants of Flash such as SLC, TLC, QLC,... Could create SLC-NAND-Flash but seems a bit too much for now.

Reply:
Updated!